custom_components/bhyve/services.yaml

```diff
@@ -43,3 +43,13 @@ set_manual_preset_runtime:
     minutes:
       description: Number of minutes to set the preset runtime
       example: 15
+
+set_smart_watering_soil_moisture:
+  description: Set the smart watering soil moisture level for a device entity
```

Comment:
Should this reference the device entity or the zone?

Reply:
I wasn't sure if I should reference what it is in home assistant or what the controller refers to it as. I just copied `set_manual_preset_runtime` which refers to it as device entity. But looking at all of the others, they refer to zone. I will update.

Reply:
`set_manual_preset_runtime` sets the value on the entire device (as far as I can tell and as far as the API parameters show).

It appears plausible that for a multi-zone controller it is possible to have landscapes with different properties (different soils, water/sun exposure etc) and so the moisture levels should be considered independent. So, I think update the description to refer to the zone :)